src/styles/index.scss

```
@@ -1,3 +1,126 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
+
+  $main-accent: #060B35;
+  $secondary: #616070;
+  $blue-accent: #00ACDC;
+  $white-accent: #fff;
+
+  body {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+
+    .card-block {
+      width: 200px;
+      min-height: 408px;
+      padding: 32px 16px 16px;
+      border: 1px solid #F3F3F3;
+      border-radius: 5px;
+      display: flex;
+      flex-direction: column;
+      position: relative;
+
+      .card-img {
+        width: 160px;
+        height: 134px;
+        margin-left: 3px;
+      }
+      .card-title {
+        margin-top: 40px;
+        margin-bottom: 4px;
+        font-weight: 500;
+        font-size: 12px;
+      }
+
+      .card-subtitle {
+        color: $secondary;
+        font-size: 10px;
+        line-height: 14px;
+        margin-bottom: 17px;
+      }
+
+      .group-one {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .stars {
+          display: flex;
+          align-items: center;
+
+          &__star {
+            display: block;
+            width: 16px;
+            height: 16px;
+            background-image: url('../images/star.svg');
+            background-repeat: no-repeat;
+            background-position: center;
+            margin-left: 4px;
+
+            &:first-child {
+              margin-left: 0;
+            }
+          }
+        }
+
+        .reviews {
+          font-size: 10px;
+          line-height: 14px;
+          color: $main-accent;
+        }
+
+        .stars--4 .stars__star:nth-child(-n+4) {
+          background-image: url('../images/star-active.svg');
+        }
+      }
+
+      .wrapper-price {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        margin-top: 23px;
+
+        .price-title {
+          color: $secondary;
+        }
+
+        .price {
+          font-size: 16px;
+          font-weight: 700;
+        }
+      }
+
+      .button-buy {
+        width: 166px;
+        height: 40px;
+        border-radius: 5px;
+        background-color: $blue-accent;
+        border: 1px solid $blue-accent;
+        color: $white-accent;
+        font-size: 14px;
+        line-height: 16px;
+        font-weight: 700;
+        cursor: pointer;
+        position: absolute;
+        bottom: 16px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-decoration: none;
+        transition: 0.3s ease-in-out;
+        text-transform: uppercase;
+
+        &:hover {
+          background-color: $white-accent;
+          color: $blue-accent;
+          transition: 0.3s ease-in-out;
```

dont need duplicate transition rule 
```suggestion
        transition: 0.3s ease-in-out;
        text-transform: uppercase;

        &:hover {
          background-color: $white-accent;
          color: $blue-accent;
```